Building from source requires latex

While updating the nix package I noticed that building from source now requires latex. However the error message wasn't very intuitive:

```
Making all in doc
  make[1]: Entering directory '/tmp/nix-build-cddlib-0.94j.drv-0/source/doc'
  make[1]: *** No rule to make target 'cddlibman.pdf', needed by 'all-am'.  Stop.
```

And the dependency is not mentioned in the README's building from source section.

--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -3,35 +3,46 @@
 # We do not want to make latex & pdflatx & dvips requirements for building
 # cddlib, so we ship the built manual as part of the distribution.
 
-if LATEX
 LATEX_SRC = $(srcdir)/cddlibman.tex $(srcdir)/cddlibman.bbl
 
 # Building twice seems to be enough to get references right.
 cddlibman.dvi: ${LATEX_SRC}
+if LATEX
 	$(latex) $<
 	$(latex) $<
+else
+	@echo "LaTeX was not found. Please build from the released source tarball or install LaTeX and rerun ./configure."
+	@false
+endif
 
-if PDF
 cddlibman.pdf: ${LATEX_SRC}
+if PDF
 	$(pdflatex) $<
 	$(pdflatex) $<
+else
+	@echo "PdfLatex was not found. Please build from the released source tarball or install PdfLatex and rerun ./configure."
+	false
 endif
 
-if PS
 cddlibman.ps: cddlibman.dvi
+if PS
 	$(dvips) cddlibman.dvi
+else
+	@echo "dvips was not found. Please build from the released source tarball or install dvips and rerun ./configure."
+	false
 endif
 
-if HTML
 # Wipe the HTML output directory as latex2html's perl calls otherwise try to
 # find images.pl in there without putting cddlibman into @INC.
 # We depend on cddlibman.dvi so we know that the references have been correctly
 # resolved in the .aux file.
 cddlibman/cddlibman.html: ${LATEX_SRC} html.sty cddlibman.dvi
+if HTML
 	rm -rf cddlibman
 	$(latex2html) $<
-endif
-
+else
+	@echo "latex2html was not found. Please build from the released source tarball or install latex2html and rerun ./configure."
+	false
 endif
 
 # Include the PDF, PS, and DVI documentation in the distributed tarball.  Note

